app/models/reason.rb

```diff
@@ -0,0 +1,3 @@
+class Reason < ActiveRecord::Base
```

Comment:
I seriously think we should have this model. `Reason` is very generic and will be very hard to follow in the future when looking at the codebase. Also it means extra SQL queries when working with `EvidenceCheck`. 

If you insist on having a dedicated table for this 1 field, let's at least rename it to something like `EvidenceCheckInvalidReason` or something between those lines, so it's clear what this is for.